website/templates/log_templates.mako

```diff
@@ -191,6 +191,19 @@ invisible on
   {{/ifnot}}
 </script>
 
+<script type="text/html" id="addon_file_renamed">
+  {{#if params.source.materialized.endsWith('/')}}
```

Comment:
There is still code duplicate in this if and if not block, the real difference is only one line <span class="overflow log-folder">{{ params.destination.materialized }}</span>, you can reduce the if block just contains that part.